README.md

```diff
@@ -10,127 +10,125 @@ neural network (GNN) that predicts an association to a target molecule, e.g., a
 DeepFPlearn<sup>+</sup> is an extension of deepFPlearn[[2]](#2), which uses binary fingerprints to represent the
 molecule's structure computationally.
 
-## Setting up Python environment
+## Installation
 
 The DFPL package requires a particular Python environment to work properly.
 It consists of a recent Python interpreter and packages for data-science and neural networks.
-The exact dependencies can be found in the
-[`requirements.txt`](requirements.txt) (which is used when installing the package with pip)
-and [`environment.yml`](environment.yml) (for installation with conda).
 
 You have several ways to provide the correct environment to run code from the DFPL package.
 
-1. Use the automatically built docker/Singularity containers
-2. Build your own container [following the steps here](container/README.md)
-3. Setup a python virtual environment
-4. Set up a conda environment install the requirements via conda and the DFPL package via pip
+1. Use conda (bioconda) to install the package
+2. Set up a Python virtual environment
+3. Use the automatically built Docker
+4. Use the automatically built Singularity containers
 
-In the following, you find details for option 1., 3., and 4.
+### Conda (bioconda)
+
+The package is also available on Bioconda. You can find the Bioconda recipe here and
+[![install with bioconda](http://bioconda.github.io/recipes/deepfplearn/README.html)]
+
+First create an environment with the following command:
+
+```shell
+conda create --override-channels --channel conda-forge --channel bioconda  -n dfpl python3.8 deepfplearn 
+```
+
+If you have a GPU available you can install the package with additional tensorflow-gpu package:
+
+```shell
+conda create --override-channels --channel conda-forge --channel bioconda  -n dfpl python3.8 deepfplearn tensorflow-gpu==2.9.3
+```
+
+Then activate the environment:
+
+```shell
+conda activate dfpl
+```
+
```

Comment:
Now, you can start using `deepFPlearn` as described in section [Running deepFPlearn](README.md#running-deepfplearn)